CHANGES.rst

```diff
@@ -1,3 +1,6 @@
+1.0
```

Comment:
longer term it would be nice to have a little more detailed changelog, e.g. dates for the releases, maybe even reference back to the PR numbers.
(but this is not an action item for this PR).